### PR TITLE
HW-52639 default to wallet country for tmc selection

### PR DESCRIPTION
**Pull request by peter-joseph**

### CODE REVIEW

File: ui/src/main/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodPresenter.java [hunk 1]
```diff
@@ -69,16 +77,41 @@ public void onKeysLoaded(@Nullable final HyperwalletTransferMethodConfigurationK
...
+
+                                if (country == null) { // param and user country is null
+                                    country = key.getCountry(DEFAULT_COUNTRY_CODE);
+                                }
+
+                                String currencyCodeString = currencyCode;
+                                Set<Currency> currencies = key.getCurrencies(country.getCode());
+                                if (TextUtils.isEmpty(currencyCodeString) && currencies != null
+                                        && !currencies.isEmpty()) {
+                                    currencyCodeString = ((Currency) currencies.toArray()[0]).getCode();
```
Comment (vshcherbyna-epam) [inline ui/src/main/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodPresenter.java]: Could you clarify conversion to an array there, as far as I know Sets are unordered. We may expect unpredictable results there.

File: ui/src/test/resources/user_ca_response.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "country": "CA",
+  "postalCode": "H0H0H0",
+  "language": "en",
+  "programToken": "prg-83836cdf-2ce2-4696-8bc5-f1b86077238c",
+  "links": [
+    {
+      "params": {
+        "rel": "self"
+      },
+      "href": "https://api.sandbox.hyperwallet.com/rest/v3/users/usr-f9154016-94e8-4686-a840-075688ac07b5"
```
Comment (azakrevska-epam) [inline ui/src/test/resources/user_ca_response.json]: @peter-joseph Could you change real link :
https://localhost:8181/rest/v3/users/usr-123

File: ui/src/test/resources/user_no_country_response.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+  "token": "usr-f9154016-94e8-4686-a840-075688ac07b5",
+  "status": "PRE_ACTIVATED",
+  "createdOn": "2017-10-30T22:15:45",
```
Comment (azakrevska-epam) [inline ui/src/test/resources/user_no_country_response.json]: @peter-joseph Could you change 
token to "usr-123", 
created on to "2019-05-23T22:15:45"
program token to "prg-234"
and href to local host?

File: ui/src/test/resources/user_not_configured_country_response.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
+{
+  "token": "usr-f9154016-94e8-4686-a840-075688ac07b5",
```
Comment (azakrevska-epam) [inline ui/src/test/resources/user_not_configured_country_response.json]: see previous comment

**Pull request merged**